src/Microsoft.DotNet.Build.Tasks/PackageFiles/CodeCoverage.targets

```diff
@@ -6,7 +6,6 @@
   -->
   <PropertyGroup>
     <OpenCoverVersion>4.6.519</OpenCoverVersion>
-    <CoverletMsbuildVersion>2.3.0</CoverletMsbuildVersion>
```

Comment:
There is a corresponding change in CoreFx, removing the properties at: https://github.com/dotnet/corefx/blob/d6646295b889daf9cf46caae1f9097b719873439/external/test-runtime/XUnit.Runtime.depproj#L22